### PR TITLE
Adds fireplaces, eswords are hot, cardboard is flammable

### PR DESCRIPTION
**Pull request by coiax**
This does #19115 but with the fixes that need to happen.

:cl: coiax, Niknakflak
add: Added fireplaces, fuel with logs or paper and set on fire and enjoy that warm comforting
glow.
fix: Cardboard cutouts are now flammable.
add: Cigarettes, fireplaces and candles now share a pool of possible igniters. So you can now esword that candle on, but turn it on first.
add: Eswords will ignite plasma in the air if turned on.
/:cl:
- Fireplaces now use world.timer
- Fireplaces no longer prompt input() for inserting logs, it just
  takes as many logs as possible
- Paper and paper bins can be thrown on the fire, thirty paper is worth
  one log of burn time.
- One log gives 15 seconds of burn time, the fireplace can hold up to 5
  minutes of fuel.
- Ignitable items now use a /obj level proc to generate their messages,
  currently using this are cigarettes, candles, fireplaces
- The fireplace can be put out with an extinguisher
- Cardboard cutouts are now flammable
- The fireplace is only "warm and cozy" when lit
- Paperbins qdel their stored papers when destroyed (probably did that
  already, but no harm in making sure)
- Also removed some returns hanging around

### CODE REVIEW

File: code/game/objects/objs.dm [hunk 1]
```diff
@@ -220,4 +220,26 @@
 /obj/proc/CanAStarPass()
 	. = !density
 
-
+/obj/proc/ignite_with_item_message(obj/item/O, mob/user)
+	var/lighting_text = null
+	if(istype(O, /obj/item/weapon/weldingtool))
```
Comment (ChangelingRain) [inline code/game/objects/objs.dm]: what the fuck coiax

**Comment by Supermichael777** [comment]
leave the snowflake messages to fukn cigs man.

**Comment by coiax** [comment]
Who doesn't want to light a candle by swinging an esword

**Comment by Niknakflak** [comment]
This is better than my PR and everyone knows it.  It sucks that you are making the logs burn for five fucking minutes though.  There was a reason it was at 15 or so.  Either way, please stop touching everything I touch, I try to avoid you.

**Comment by coiax** [comment]
Logs don't give any more time than your PR, just the maximum fuel capacity has been increased from one minute to five. Also, given that yours used byond ticks, and this uses world.time, it's more like your original max fuel time was around one and a half to two minutes.

Making them constructable and movable is for another PR I think.

**Comment by Niknakflak** [comment]
Xhuis said it was like 15 minutes.  Im wrong then and sorry for stating so.  This is fine and better.

**Comment by Niknakflak** [comment]
:+1:

**Pull request merged**